src/pseudo/PspUpf.jl

```diff
@@ -65,164 +65,169 @@ Does not support:
 - Projector-augmented wave potentials
 - GIPAW reconstruction data
 """
-function PspUpf(path; identifier=path)
+function PspUpf(path; identifier=path, rcut=nothing)
     pseudo = load_upf(path)
 
     unsupported = []
-    pseudo["header"]["has_so"]               && push!(unsupported, "spin-orbit coupling")
-    pseudo["header"]["pseudo_type"] == "SL"  && push!(unsupported, "semilocal potential")
-    pseudo["header"]["pseudo_type"] == "US"  && push!(unsupported, "ultrasoft")
+    pseudo["header"]["has_so"] && push!(unsupported, "spin-orbit coupling")
+    pseudo["header"]["pseudo_type"] == "SL" && push!(unsupported, "semilocal potential")
+    pseudo["header"]["pseudo_type"] == "US" && push!(unsupported, "ultrasoft")
     pseudo["header"]["pseudo_type"] == "PAW" && push!(unsupported, "projector-augmented wave")
-    pseudo["header"]["has_gipaw"]            && push!(unsupported, "gipaw data")
+    pseudo["header"]["has_gipaw"] && push!(unsupported, "gipaw data")
     pseudo["header"]["pseudo_type"] == "1/r" && push!(unsupported, "Coulomb")
     length(unsupported) > 0 && error("Pseudopotential contains the following unsupported" *
                                      " features/quantities: $(join(unsupported, ","))")
 
-    Zion        = Int(pseudo["header"]["z_valence"])
-    rgrid       = pseudo["radial_grid"]
-    drgrid      = pseudo["radial_grid_derivative"]
-    lmax        = pseudo["header"]["l_max"]
-    vloc        = pseudo["local_potential"] ./ 2  # (Ry -> Ha)
+    Zion = Int(pseudo["header"]["z_valence"])
+    rgrid = pseudo["radial_grid"]
+    drgrid = pseudo["radial_grid_derivative"]
+    lmax = pseudo["header"]["l_max"]
+    vloc = pseudo["local_potential"] ./ 2  # (Ry -> Ha)
     description = get(pseudo["header"], "comment", "")
```

Comment:
This was formatted so nicely ... don't break that for no reason.

Reply:
<img width="596" alt="image" src="https://github.com/JuliaMolSim/DFTK.jl/assets/75646713/035de37b-f34e-49fc-af8b-245adc182bc7">
Sorry!